ytemplates/bootstrap/value.spam_protection.tpl.php

```diff
@@ -15,6 +15,7 @@
             overflow: hidden;
             height: 1px;
             opacity: 1;
+	    padding-top: 1px;
```

Comment:
```suggestion
            padding-top: 1px;
```
Leerzeichen statt Tabs